scripts/restyle.sh

```diff
@@ -4,19 +4,25 @@ FILES=`find ${DIRNAME} -name "*.cpp" -or -name "*.hpp"`
 
 
 OPTIONS="
---style=kr \
```

Comment:
I would like to follow the **Kernighan & Ritchie** style. USBGuard uses the same.